mmv1/third_party/terraform/tests/resource_composer_environment_test.go.erb

```diff
@@ -397,6 +396,39 @@ func TestAccComposerEnvironment_ComposerV2(t *testing.T) {
     },
   })
 }
+
```

Comment:
This test should only run in `beta` could you add the <% unless ga %> flags to gate the code accordingly

Reply:
Nevermind, already gated